freqtrade/exchange/exchange.py

```diff
@@ -877,42 +877,67 @@ def get_dry_market_fill_price(self, pair: str, side: str, amount: float, rate: f
 
         return rate
 
-    def _is_dry_limit_order_filled(self, pair: str, side: str, limit: float) -> bool:
+    def _fill_dry_limit_order(self, pair: str, side: str,  # noqa: max-complexity: 13
```

Comment:
Whenever you need to `noqa` something, then that's a warning sign that it should be done differently - maybe split into multiple functions.

The function itself doesn't look "that" complex though - are you sure this is really needed in it's current state?

Reply:
It's not need now.